crates/cubecl-wgpu/src/compiler/spirv.rs

```diff
@@ -344,7 +380,7 @@ fn dump_spirv(compiled: &CompiledKernel<VkSpirvCompiler>, name: &str, id: cubecl
         hash::{DefaultHasher, Hash, Hasher},
     };
 
-    if let Ok(dir) = std::env::var("CUBCEL_DEBUG_SPIRV") {
+    if let Ok(dir) = std::env::var("CUBECL_DEBUG_SPIRV") {
```

Comment:
I feel like at some point it would be very nice to have a revamp `DebugLogger` where we would save stuff in a database file as well. Just an idea.

Reply:
Yeah I was thinking about this and couldn't come up with a great way to handle it, so this was my temporary fix